Makefile

```diff
@@ -44,6 +46,9 @@ start-dockerdemo:
 clean-dockerdemo:
 	scripts/dockerhost/cleanup-dockerhosts
 
+ssh:
```

Comment:
do we need to do `vagrant up` or `make demo` first? Otherwise this target will fail right?